src/SmoFacade/Database.cs

```diff
@@ -40,6 +40,17 @@ public void RestoreWithRecovery()
       _database.Refresh();
     }
 
+    public void DropAssociatedLogins()
+    {
+      var logins = Users.Where(u => u.Login != null && u.Login.Name != "sa")
```

Comment:
Small nit pick, can we pls place `"sa"` in a variable called `SystemAdminAccountName` or something like that?
Just feel it would be easier to understand what it is for engineers not familiar with SQL Server logins.

I had to google what "sa" was when reading through this 😅